amd_openvx_extensions/amd_media/kernels.cpp

```diff
@@ -95,7 +95,6 @@ vx_status initialize_ffmpeg()
 		initialized = true;
 		av_log_set_callback(av_log_callback);
 		av_log_set_level(AV_LOG_ERROR);
-		av_register_all();
```

Comment:
why do you need to remove this?

Reply:
Sorry about that looks I missed adding a description for that change.
[Here](https://github.com/FFmpeg/FFmpeg/blob/70d25268c21cbee5f08304da95be1f647c630c15/doc/APIchanges#L86) is the changelog for ffmpeg and we can see that the function is deprecated.